pkg/storage/trb_request_attendee.go

```diff
@@ -150,45 +174,52 @@ func (s *Store) DeleteTRBRequestAttendee(ctx context.Context, id uuid.UUID) (*mo
 	return &deleted, err
 }
 
-// GetAttendeeComponentByEUA attempts to retrieve the component of a given EUA user ID and TRB Request ID
-func (s *Store) GetAttendeeComponentByEUA(ctx context.Context, euaID string, trbRequestID uuid.UUID) (*string, error) {
+// GetAttendeeByEUAIDAndTRBID attempts to retrieve an attendee of a given EUA user ID and TRB Request ID
+func (s *Store) GetAttendeeByEUAIDAndTRBID(ctx context.Context, euaID string, trbRequestID uuid.UUID) (*models.TRBRequestAttendee, error) {
```

Comment:
I figured this was more useful and that specific resolver can return the single field.